src/main.h

```diff
@@ -174,7 +177,8 @@ void UnregisterNodeSignals(CNodeSignals& nodeSignals);
  * @param[out]  dbp     If pblock is stored to disk (or already there), this will be set to its location.
  * @return True if state.IsValid()
  */
-bool ProcessNewBlock(CValidationState &state, CNode* pfrom, CBlock* pblock, bool fForceProcessing, CDiskBlockPos *dbp);
+bool ProcessNewBlock(CValidationState &state, CNode* pfrom, CBlock* pblock, bool fForceProcessing, CDiskBlockPos *dbp,
+libzcash::ProofVerifier& verifier = disabledVerifier);
```

Comment:
Probably a good point. Seems like I did forgot to pass the verifier in line 3861 in main.cpp to ProcessNewBlock so it may not have been verifying indeed :) (Or just verifying an empty batch in line 3881)